### PR TITLE
Lay out /work/listings/:id

### PR DESCRIPTION
**Pull request by woeltjen**
## Description
Lay out the contents of the Listing page.

## How to Test
1. Navigate to /work/listings/95_vic (or some other listing)
2. Verify that major page elements are laid out sensibly
    1. Image gallery at top
    2. Listing details below (to left, in desktop, or at center, on tablet/mobile)
    3. Booking Card to the right (or at bottom, on tablet/mobile)

Note that individual page elements are still unstyled (subsequent PRs!)

Which devices did you test on?
- [x] Chrome on Mac
- [ ] Chrome on PC
- [ ] Firefox on Mac
- [ ] Firefox on PC
- [ ] Safari iPhone
- [ ] Chrome Android

## REVIEWERS:
Check against these principles:

### High level
Does this code need to be written?
What are the alternatives?
Will this implementation become a support issue?
How much error margin does this solution have?

### Code
* Does the code follow industry standards?
JS: https://github.com/airbnb/javascript
React: https://github.com/airbnb/javascript/tree/master/react
https://github.com/vasanthk/react-bits
Documentation headers: http://usejsdoc.org/index.html

* Is there duplicated code? Can it be refactored into a shared method?
* Is the code consistent with our project?
* Are there unit tests? Do they test the states?
* Is the person refactoring another developer's code? If possible, did the original developer approve?

### Variables/Naming:
* Would the variable type led to future edge cases?
* Are the variable naming clear? Would the value contain something other than what the name describes.

### Security
* Can this be hacked or abused by the user?

## Further Work
1. Gallery layout (move button)
2. View Photos functionality
3. Contact Host functionality
4. Booking Card layout
5. Booking Card functionality

### CODE REVIEW

**Comment by woeltjen** [comment]
This seems like it's in a reasonable state to review. One thing worth noting is that the BookingBar will obscure part of the footer on tablet/mobile. This is actually the case for the old design, too, so this is at parity, but raising it in case anyone sees a solution

File: src/pages/listing/ListingPage.tsx [hunk 1]
```diff
@@ -1,14 +1,31 @@
...
 
 import { Listing } from 'networking/listings';
 
-const ListingPage = (listing: Listing) => (<>
-  <ListingGallery {...listing} />
-  <ListingInformation {...listing} />
-  <BookingCard {...listing} />
-</>);
+const ListingPage = (listing: Listing) => (
+  <>
```
Comment (kcvan) [inline src/pages/listing/ListingPage.tsx]: I think this can be <Fade>

Comment (tc) [reply src/pages/listing/ListingPage.tsx]: @kcvan  your comment got cut off

Comment (kcvan) [reply src/pages/listing/ListingPage.tsx]: ah thanks, was going to say this can be `<Fade>`

File: src/pages/listing/ListingPage.tsx [hunk 2]
```diff
@@ -9,7 +9,7 @@ import BookingBar from './BookingBar';
 import { Listing } from 'networking/listings';
 
 const ListingPage = (listing: Listing) => (
-  <>
+  <Fade>
```
Comment (tc) [inline src/pages/listing/ListingPage.tsx]: sometimes it's not good to use a `Fade` effect. I think it's fine here.

ie, if we want the page to load immediately, like the signup/booking pages. Anything slowing the user down to conversion is not good. cc @kcvan
